omod-common/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/controller/BaseRestController.java

```diff
@@ -83,6 +84,18 @@ public SimpleObject validationExceptionHandler(ValidationException validationExc
 		return RestUtil.wrapValidationErrorResponse(validationException);
 	}
 	
+	/**
+	 * Handle ConvertionException - return response with exception message - ConversionUtil throws
+	 * ConversionException
+	 */
+	@ExceptionHandler(ConversionException.class)
+	@ResponseBody
+	public SimpleObject conversionExceptionHandler(ConversionException conversionException, HttpServletRequest request,
+	        HttpServletResponse response) {

```

Comment:
This seems to address the issue, there is existing code that did this except that it didn't send back the error message, may be it was intentional not to do so since error messages can contain sensitive data you might want to sent back to any user?